client-v2/src/shared/selectors/shared.ts

```diff
@@ -182,6 +182,18 @@ const createCollectionStageGroupsSelector = () => {
   );
 };
 
+const createPreviouslyModifiedCollectionSelector = () => {
```

Comment:
is `createPreviouslyLiveArticlesInCollectionSelector` more descriptive?

Reply:
updated.